share/picongpu/dockerfiles/README.rst

```diff
@@ -25,7 +25,7 @@ This exposes the ISAAC port to connect via the webclient to.
 .. code:: bash
 
     docker pull ax3l/picongpu
-    docker run --runtime=nvidia -p 2459:2459 -t ax3l/picongpu:0.5.0 lwfa_live
+    docker run --runtime=nvidia -p 2459:2459 -t ax3l/picongpu:0.5.0-rc1 lwfa_live
```

Comment:
`ax3l` repo is still used

Reply:
Please tell me if you (e.g. @PrometheusPi and @psychocoderHPC ) create a dockerhub account.
I already reserved a `picongpu/` org a while back and will then add you to it :)
https://hub.docker.com/orgs/picongpu

Reply:
"Register now and increase the bus-factor by a three-fold."™

Reply:
@ax3l I am now registered at docker.com as `psychocoderhpc`